tests/v2/validation/publicapi/clusterroletemplatebindings/README.md

```diff
@@ -0,0 +1,20 @@
+# ClusterRoleTemplateBinding(v1)
+This repository contains Golang automation tests for the ClusterRoleTemplateBinding functionality by SteveAPI(V1) and kubectl. This is part of Public API functionality to generate Custom Resource Definitions (CRD's) for ClusterRoleTemplateBinding. ClusterRoleTemplateBindings is allowing users to perform all the CRUD operations by using SteveAPI(V1) OR kubectl. These automation tests contains both success and error test cases to make sure the functionality is working as expected.
+
+## Pre-requisites
+- Ensure you have an existing cluster that the user has access to. If you do not have a downstream cluster in Rancher, create one first before running this test.
+- Some tests require creating additional downstream cluster. Providing the provisioningInput parameter with appropriate values is mandatory unless you are skipping those tests.
```

Comment:
Das,  Looks like a copy paste error. I don't see any tests that require creating additional downstream clusters. We can remove this